app/ts/ReportingUI/components/Reporting.tsx

```diff
@@ -24,32 +21,87 @@ export const DisplayExtraInfo = ({ marketData }: MarketProps) => {
 				readOnly = { true }
 			/>
 		</>
 	}
 	return <>
-		<p> Description: { marketData.deepValue.parsedExtraInfo.description }</p>
-		<p> Long Description: { marketData.deepValue.parsedExtraInfo.longDescription }</p>
-		<p> Categories: { (marketData.deepValue.parsedExtraInfo.categories || []).join(', ') }</p>
-		<p> Tags: { (marketData.deepValue.parsedExtraInfo.tags || []).join(', ') }</p>
+		<span> <b>Description:</b> { marketData.deepValue.parsedExtraInfo.description }</span>
+		<span> <b>Long Description:</b> { marketData.deepValue.parsedExtraInfo.longDescription }</span>
+		<span> <b>Categories:</b> { (marketData.deepValue.parsedExtraInfo.categories || []).join(', ') }</span>
+		<span> <b>Tags:</b> { (marketData.deepValue.parsedExtraInfo.tags || []).join(', ') }</span>
 	</>
 }
 
 export const Market = ({ marketData }: MarketProps) => {
 	if (marketData.deepValue === undefined) return <></>
-	return <div>
-		<p> Market Address: { marketData.deepValue.marketAddress }</p>
-		<p> Market Creator: { marketData.deepValue.marketCreator }</p>
-		<p> Market Type: { marketData.deepValue.marketType }</p>
-		<p> Recommended Trade Interval: { marketData.deepValue.recommendedTradeInterval }</p>
-		<p> Display Prices: { marketData.deepValue.displayPrices.join(', ') }</p>
-		<p> Outcomes: { marketData.deepValue.outcomes.join(', ') }</p>
+
+	const formatVolumes = () => {
+		if (marketData.deepValue === undefined) return ''
+		const marketType = MARKET_TYPES[marketData.deepValue.hotLoadingMarketData.marketType]
+		const volumes = marketData.deepValue.hotLoadingMarketData.outcomeVolumes
+		switch(marketType) {
+			case 'Categorical':
+			case 'Scalar': {
+				return volumes.join(', ')
+			}
+			case 'Yes/No': {
+				return <div style = 'display: grid'>
+					{ YES_NO_OPTIONS.map((option, index) => (
+						<span>{ option }: { volumes[index] === undefined ? 'undefined' : bigintToRoundedPrettyDecimalString(volumes[index], 18n, 4) } DAI</span>
+					)) }
+				</div>
+			}
+			case undefined: throw new Error(`invalid marketType: ${ marketData.deepValue.hotLoadingMarketData.marketType }`)
+			default: assertNever(marketType)
+		}
+	}
+	const formatWinningOption = () => {
+		if (marketData.deepValue === undefined) return ''
+		const marketType = MARKET_TYPES[marketData.deepValue.hotLoadingMarketData.marketType]
+		const payouts = marketData.deepValue.hotLoadingMarketData.winningPayout
+		switch(marketType) {
+			case 'Categorical':
+			case 'Scalar': {
+				return payouts.join(', ')
+			}
+			case 'Yes/No': {
+				const winningIndex = payouts.findIndex((payout) => payout > 0)
+				return YES_NO_OPTIONS[winningIndex]
+			}
+			case undefined: throw new Error(`invalid marketType: ${ marketData.deepValue.hotLoadingMarketData.marketType }`)
+			default: assertNever(marketType)
+		}
+	}
+
+	return <div style = 'display: grid'>
+		<span> <b>Market Address:</b> { marketData.deepValue.marketAddress }</span>
```

Comment:
Related to style comment above, this one has space before/after bold tag, but not around span tag, which seems arbitrary.

Reply:
yeah, I unify them in future pr